index.js

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+import dotenv from "dotenv";
+import greetRoutes from "./routes/public/greet.js";
+
+dotenv.config();
+const app = express();
+
+const PORT = process.env.PORT;
```

Comment:
const PORT = process.env.PORT || 8000

Replace it in such a way.

Reply:
Done